src/data/token.js

```diff
@@ -36,31 +37,27 @@ function createKey() {
 }
 
 function get(key) {
-  return sessionStorage.getItem(key) || qsData[key]
+  return retrieve(key) || qsData[key]
 }
 
-// TODO: consider moving this to localStorage so new tabs won't force re-authn.
 function set(key, defaultValue = '') {
```

Comment:
If we can narrow down the scope of `set()` so that it's single responsible is to store what it's given, I think that will help readability here. Currently it appears to go looking various places to find the value that it should store... which feels like a not-dependency-injected kind of code-smell. 

However, this is out of scope for the current PR, so feel free to just make a note to revisit this later.

Reply:
it's great to have fresh eyes, thanks for pointing this out.  https://github.com/silinternational/wecarry-ui/pull/129/commits/29dccbee7f5261fcb6b9c5cedfe9e5ba2af7bba5